homeassistant/components/tahoma/light.py

```diff
@@ -0,0 +1,133 @@
+"""Tahoma light platform that implements dimmable tahoma lights."""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_EFFECT,
+    SUPPORT_BRIGHTNESS,
+    SUPPORT_EFFECT,
+    LightEntity,
+)
+from homeassistant.const import STATE_OFF, STATE_ON, STATE_UNKNOWN
+
+from . import DOMAIN as TAHOMA_DOMAIN, TahomaDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=30)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the Tahoma light platform."""
+    if discovery_info is None:
+        return
+    controller = hass.data[TAHOMA_DOMAIN]["controller"]
+    devices = []
+
+    for device in hass.data[TAHOMA_DOMAIN]["devices"]["light"]:
+        devices.append(TahomaLight(device, controller))
+
+    async_add_entities(devices, True)
+
+
+class TahomaLight(TahomaDevice, LightEntity):
+    """Representation of a Tahome light."""
+
+    def __init__(self, tahoma_device, controller):
+        """Initialize the device."""
+        super().__init__(tahoma_device, controller)
+        self._skip_update = False
+        self._effect = None
+        self._state = STATE_UNKNOWN
```

Comment:
```suggestion
        self._state = None
```

entity.py will set this to `STATE_UNKNOWN` if its none
`            state = STATE_UNKNOWN if state is None else str(state)`